src/components/suggest_item/suggest_item.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { EuiIcon, IconPropType } from '../icon';
+
+const colorToClassNameMap = {
+  tint0: 'euiSuggestItem__type--tint0',
+  tint1: 'euiSuggestItem__type--tint1',
+  tint2: 'euiSuggestItem__type--tint2',
+  tint3: 'euiSuggestItem__type--tint3',
+  tint4: 'euiSuggestItem__type--tint4',
+  tint5: 'euiSuggestItem__type--tint5',
+  tint6: 'euiSuggestItem__type--tint6',
+  tint7: 'euiSuggestItem__type--tint7',
+  tint8: 'euiSuggestItem__type--tint8',
+  tint9: 'euiSuggestItem__type--tint9',
+};
+
+export const COLORS = Object.keys(colorToClassNameMap);
+
+const labelDisplayToClassMap = {
+  fixed: 'euiSuggestItem__labelDisplay--fixed',
+  expand: 'euiSuggestItem__labelDisplay--expand',
+};
+
+export const DISPLAYS = Object.keys(labelDisplayToClassMap);
+
+export const EuiSuggestItem = ({
+  className,
+  label,
+  type,
+  labelDisplay,
+  description,
+  ...rest
+}) => {
+  const classes = classNames('euiSuggestItem', className);
+
+  let colorClass = '';
+
+  const labelDisplayClass = classNames(
+    'euiSuggestItem__label',
+    labelDisplayToClassMap[labelDisplay],
+    {
+      'euiSuggestItem__labelDisplay--expand': !description,
+    }
+  );
+
+  if (type && type.color) {
+    if (COLORS.indexOf(type.color) > -1) {
+      colorClass = colorToClassNameMap[type.color];
+    }
+  }
+
+  return (
+    <div className={classes} {...rest}>
+      <span className={`euiSuggestItem__type ${colorClass}`}>
+        <EuiIcon type={type.iconType} />
+      </span>
+      <span className={`euiSuggestItem__label ${labelDisplayClass}`}>
```

Comment:
You no longer need `euiSuggestItem__label` here because you've added it to `labelDisplayClass` at line 41.

<img width="513" alt="Screen Shot 2019-07-18 at 18 13 07 PM" src="https://user-images.githubusercontent.com/549577/61495695-b5f62f80-a987-11e9-927a-912ef03ca8fe.png">


```suggestion
      <span className={labelDisplayClass}>
```